core/include/prometheus/histogram.h

```diff
@@ -68,10 +70,10 @@ class PROMETHEUS_CPP_CORE_EXPORT Histogram {
   ClientMetric Collect() const;
 
  private:
-  const BucketBoundaries bucket_boundaries_;
+  BucketBoundaries bucket_boundaries_;
   mutable std::mutex mutex_;
   std::vector<Counter> bucket_counts_;
-  Gauge sum_;
+  Gauge sum_{};
```

Comment:
I think you could omit the `{}` here. The `sum_` member was only present in the member initializer list to show the order of members.